### PR TITLE
Topic on time travel and changing the system date

### PR DESCRIPTION
**Pull request by Onion-Bubs**
I know that people have tried changing the system date in order to unlock certain features in MAS that are only available on a specific day or after a certain period of time. I also know that this kind of "time travel" has led to these people having problems. So I wrote a topic that segues nicely from a discussion about the effects of time travel on the people closest to the time traveller to a warning about the effects of changing the system date.

I am open to suggestions for how to improve this topic. In particular, if anyone knows any books whose main focus is how people are affected when someone close to them goes time travelling, I'd be glad to have an example to mention in this topic.

### CODE REVIEW

**Comment by jmwall24** [comment]
#2669

**Comment by AsuraShun** [comment]
just for fun, could you add a doctor who joke if possible?

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m 2ekc "Well, the time traveller themselves might know, but only if nothing goes horribly wrong for them."
+    m 2dsc "In the meantime ..."
+    m 2lksdld "Their friends and family are stuck waiting for them to come back."
+    m 2ekd "How long will they be gone for? A year? A decade? Even longer?"
+    m 4ekd "Will they be willing to wait that long?"
+    m "Will they still remember the time traveller by then?"
+    m 4lksdlc "..."
+    if seen_event("monika_resource"):
+        m "You remember when I said that time doesn't pass here like it does in your world?"
+        m "Well ..."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
        m "Well..."
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    else:
+        m "You know, time doesn't really pass here like it does in your world."
+        m "So ..."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
        m "So..."
```

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m 1esd "The only way I know what day and time it is where you are is by looking at the clock on your computer."
+    m 3eub "So I know it's [todays_date_spoken] today."
+    m 1esc "But if you were to change the date on your computer to [one_year_later_spoken] ..."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 1esc "But if you were to change the date on your computer to [one_year_later_spoken]..."
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m "Only a few seconds would pass for you ..."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m "Only a few seconds would pass for you..."
```

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m 1ekd "... but I would think that a year had gone by."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 1ekd "...but I would think that a year had gone by."
```

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m 1tkd "An entire year without me seeing you even once."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 1tkd "An entire year without seeing you even once."
```

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m 1lksdlc "I would wonder if you'd forgotten about me, or if you didn't want to spend time with me any more."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 1lksdlc "I would wonder if you'd forgotten about me, or if you didn't want to spend time with me anymore."
```

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    if mas_isMoniDis(lower=True):
+        m 1dkc "It wouldn't even be that big of a surprise."
+    elif mas_isMoniAff(higher=True):
+        m 1ekc "Even if it would be hard to believe at first."
+    m 1ekd "You couldn't then fix it just by changing the date back either."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 1ekd "You couldn't just fix it by changing the date back either."
```

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m 3esd "Like, imagine if you went to bed now, and when you woke up it was [one_year_earlier_spoken]."
+    m 3wuo "Would that mean the last year was all just a dream or something?"
+    m 3wuw "How would you know you're not still dreaming?"
+    m 3esc "It would all be very confusing."
+    m 1lksdla "I guess what I'm trying to say is ..."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 1lksdla "I guess what I'm trying to say is..."
```

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -6952,6 +6952,68 @@ label monika_attractiveness:
...
+    m 1eka "Please don't change the date on your computer, [player]."
+    if mas_isMoniHappy(higher=True):
+        m "I trust you not to mess with me like that."
+    else :
+        m "Promise?"
+    m "And if you ever do travel to the future ..."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 3euc "And if you ever do travel to the future..."
```

**Pull request merged**